tests/data/annotation/metadata/convention/brain_rf1/patchseq_classic_metadata_test_10.tsv

```diff
@@ -0,0 +1,11 @@
+NAME	biosample_id	Date	Sample	donor_id	Mouse_date_of_birth	Mouse_age	sex	Mouse_genotype	Targeted_layer	Inferred_layer	Cre	Exon_reads	Intron_reads	Intergenic_reads	Sequencing_batch	Number_of_genes_detected	RNA_family	RNA_type	RNA_type_confidence	RNA_type_top_3	ALM_VISp_top_3	Length_bp_	Yield_pg_µl_	User	Hold_Time_min_	Soma_depth_µm_	Soma_depth_4x_	Cortical_thickness_4x_	Cortical_thickness_µm_	Traced	Exclusion_reasons	species	species__ontology_label	disease	disease__ontology_label	organ	organ__ontology_label	library_preparation_protocol	library_preparation_protocol__ontology_label	dandi_url	bil_url
```

Comment:
Since we have URL strings for `bil_url` and `dandi_url` in this metadata file, it seems we'll need further data processing to enable the Boolean facets "Has morphology" and "Has electrophysiology" for global cell-level study search.

Omitting `has_morphology` and `has_electrophysiology` fields in the metadata file itself seems reasonable if we intend for the work in this PR as an emulation of metadata we'd receive from upstream, e.g. from the the Pipelines team.  We'll just need code elsewhere to convert `bil_url` and `dandi_url` to `has_morphology` and `has_electrophysiology`, respectively.

```suggestion
NAME	biosample_id	Date	Sample	donor_id	Mouse_date_of_birth	Mouse_age	sex	Mouse_genotype	Targeted_layer	Inferred_layer	Cre	Exon_reads	Intron_reads	Intergenic_reads	Sequencing_batch	Number_of_genes_detected	RNA_family	RNA_type	RNA_type_confidence	RNA_type_top_3	ALM_VISp_top_3	Length_bp_	Yield_pg_µl_	User	Hold_Time_min_	Soma_depth_µm_	Soma_depth_4x_	Cortical_thickness_4x_	Cortical_thickness_µm_	Traced	Exclusion_reasons	species	species__ontology_label	disease	disease__ontology_label	organ	organ__ontology_label	library_preparation_protocol	library_preparation_protocol__ontology_label	dandi_url	bil_url
```

Reply:
Agreed that we won't be storing `bil_url` or `dandi_url` in BigQuery. The conversion will be part of ingest where we internally maintain a mapping of each modality to a list of metadata column headers. For example, if another consortia maintains their morphology data in the FOO repo, their data would have a foo_url column and in ingest {"morphology": ['bil_url', 'foo_url', 'bar_lab_data']} would indicate that URLs from BIL and FOO as well as some kind of data from the Bar lab are all morphology data sources that represent a single cell morphology data file for a specific cell being ingested into SCP.

Ingest will translate the presence or absence of information in the pre-defined list of custom metadata headers and infer a boolean value for `has_morphology`. (ingest will also need to add the actual URLs to MongoDB.)

Using booleans in BigQuery for modality data is lightweight and we keep the custom headers out of the metadata convention.